### PR TITLE
Bump js sdk to a branch supporting the state fix msc4222 changes and local-echo

### PR DESCRIPTION
**Pull request by toger5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
     "livekit-client": "^2.5.7",
     "lodash": "^4.17.21",
     "loglevel": "^1.9.1",
-    "matrix-js-sdk": "matrix-org/matrix-js-sdk#8e9a04cdec0f88fc876bbbf406db55b0677f005d",
+    "matrix-js-sdk": "matrix-org/matrix-js-sdk#0098c1a37d33123db5e5c54c6292a9cca7a87ca5",
```
Comment (Half-Shot) [inline package.json:90]: You can drop my change that just landed too :) #2764

Comment (toger5) [reply package.json:90]: sure. otherwise its not really testable right?

Comment (toger5) [reply package.json:90]: Lets not merge this for now but use the netlify version to test it
(so we have the save version on develop/livekit branch)

**Comment by toger5** [comment]
We can close this since we did those changes as part of: https://github.com/element-hq/element-call/pull/2767

We will move them over to the LK branch once they are merged into the js-sdk

**Pull request closed without merging**